### PR TITLE
Use CLI for bundling

### PR DESCRIPTION
**Pull request by hirasso**
**Description**

- Switch to `@swup/cli` for bundling
- Correctly separate dependencies from dev dependencies
- Includes version bump and changelog update

**Drive-By**

- Code cleanup, more readable logging

**Checks**

- [x] The PR is submitted to the `master` branch
- [x] The code was linted before pushing (`npm run lint`)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,8 +37,9 @@
 
 - Initial Release
 
-[Unreleased]: https://github.com/swup/forms-plugin/compare/0.3.1...HEAD
+[Unreleased]: https://github.com/swup/forms-plugin/compare/0.3.2...HEAD
```
Comment (daun) [inline CHANGELOG.md:40]: We can/should remove this one here as well as it's no longer in use.

File: CHANGELOG.md [hunk 2]
```diff
@@ -33,8 +37,9 @@
...
 
+[0.3.2]: https://github.com/swup/forms-plugin/releases/tag/0.3.2
```
Comment (daun) [inline CHANGELOG.md:42]: The links here go to `forms-plugin` 🤠

File: CHANGELOG.md (hunk 1 above)
Comment (hirasso) [reply CHANGELOG.md:40]: Ah! Nice

File: CHANGELOG.md (hunk 2 above)
Comment (hirasso) [reply CHANGELOG.md:42]: OMFG 🙀🤣

**Comment by hirasso** [comment]
@daun Ready :)

**Review by daun — APPROVED** [review]

**Pull request merged**